packages/server/src/adapters/electron-ipc/expose.test.ts

```
@@ -0,0 +1,33 @@
+vi.mock('electron', () => ({
+  contextBridge: {
+    exposeInMainWorld: vi.fn(),
+  },
+  ipcRenderer: {
+    on: vi.fn(),
+    send: vi.fn(),
+  },
+}))
+
+beforeEach(() => {
+  delete (globalThis as any)['orpc:default']
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)